packages/browser-service/src/browser.context.service.ts

```diff
@@ -0,0 +1,140 @@
+import {
+  COINBASE_COMMON_CONFIG,
+  CTRL_COMMON_CONFIG,
+} from '@lidofinance/wallets-testing-wallets';
+import { ConsoleLogger } from '@nestjs/common';
+import { BrowserContext, chromium } from '@playwright/test';
+import * as fs from 'fs/promises';
+import * as os from 'os';
+import path from 'path';
+
+export type BrowserOptions = {
+  locale?: string;
+  headless?: boolean;
+  slowMo?: number;
+  args?: string[];
+  permissions?: string[];
+  httpCredentials?: {
+    username: string;
+    password: string;
+  };
+};
+
+type OptionsBrowserContext = {
+  // If contextDataDir is undefined - will be created temp dir for context data.
+  // Else contextDataDir is not undefined - will be created user dir for context data in current folder.
+  contextDataDir: string;
+  browserOptions?: BrowserOptions;
+  cookies?: ReadonlyArray<{
+    name: string;
+    value: string;
+    url?: string;
+    domain?: string;
+    path?: string;
+    expires?: number;
+    httpOnly?: boolean;
+    secure?: boolean;
+  }>;
+};
+
+const excludeExtensionForWaitingHomePage = [
```

Comment:
Could you clarify why the object and its usage `excludeExtensionForWaitingHomePage` since  that there's no such usage in the `main` code and everything appears to work without it?